src/rail_gui.cpp

```diff
@@ -1989,20 +1989,20 @@ void InitializeRailGUI()
  */
 DropDownList *GetRailTypeDropDownList(bool for_replacement, bool all_option)
 {
-	RailTypes used_railtypes = RAILTYPES_NONE;
+	RailTypes used_railtypes;
+	RailTypes avail_railtypes;
```

Comment:
Did you wanted to use that in line 2033? Otherwise, it is unused ans should be removed.

Reply:
Er, yes, indeed.